packages/backend/services/propertyService.js

```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
       });
     }
   });
+  propertiesWithinPolygons.forEach((property) => {
+    calculateTravelToWork(property);
+  });
   return [...new Set(propertiesWithinPolygons.flat())];
 };
 
-const getDistance = async (startCoordinates, endCoordinates, methodOfTravel) => {
+const getDirections = async (
+  startCoordinates,
+  endCoordinates,
+  methodOfTravel
+) => {
   const token = `?access_token=${process.env.MAPBOX_TOKEN}`;
```

Comment:
i would have only the token here and add ?access_token= to the url below
```suggestion
  const token = `${process.env.MAPBOX_TOKEN}`;
```

Reply:
yeh that's cool!